setup.py

```diff
@@ -2,7 +2,7 @@
 
 setuptools.setup(
     name="csdap_bulk_download",
-    version="1.0",
+    version="1.0.1",
     description="Bulk download tool for CSDAP orders.",
     author="NASA Impact",
     author_email="csdap@uah.edu",
```

Comment:
```suggestion
    author_email="support-csda@nasa.gov",
```
To align with the SDX: https://github.com/NASA-IMPACT/csdap-frontend/pull/667